pkgs/top-level/all-packages.nix

```diff
@@ -5619,8 +5619,8 @@ with pkgs;
     (lib.addMetaAttrs { outputsToInstall = [ "jre" ]; }
       (openjdk7.jre // { outputs = [ "jre" ]; }));
 
-  jdk8 = openjdk8 // { outputs = [ "out" ]; };
-  jre8 = lib.setName "openjre-${lib.getVersion pkgs.openjdk8.jre}"
+  jdk8 = if stdenv.isArm then oraclejdk8 else openjdk8 // { outputs = [ "out" ]; };
+  jre8 = if stdenv.isArm then oraclejre8 else lib.setName "openjre-${lib.getVersion pkgs.openjdk8.jre}"
```

Comment:
I am not sure about this one. Is it a good idea to provided the closed variant only on a single platform, where other plattforms get openjdk? Is openjdk non-functional on ARM? If you want to have a shortcut to not having to compile jdk on your device, you can still use `packageOverrides`.